### PR TITLE
provider/google: Fixed outputs in serialize application task

### PR DESCRIPTION
**Pull request by nwwebb**
Made changes to output, let me how it looks @duftler @lwander

### CODE REVIEW

File: orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/serialize/SerializeApplicationTask.groovy [hunk 1]
```diff
@@ -40,10 +40,9 @@ class SerializeApplicationTask extends AbstractCloudProviderAwareTask implements
       .toBlocking()
       .first()
     Map outputs = [
-      "notification.type"  : "destroyjob",
+      "notification.type"  : "serializeApplication",
       "kato.last.task.id"  : taskId,
-      "delete.name"        : stage.context.jobName,
-      "delete.region"      : stage.context.region,
+      "delete.application" : stage.context.applicationName,
```
Comment (lwander) [inline orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/serialize/SerializeApplicationTask.groovy]: Probably shouldn't be `delete` here

Comment (lwander) [inline orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/serialize/SerializeApplicationTask.groovy]: Instead `serialize.application`

File: orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/serialize/SerializeApplicationTask.groovy [hunk 2]
```diff
@@ -40,11 +40,10 @@ class SerializeApplicationTask extends AbstractCloudProviderAwareTask implements
       .toBlocking()
       .first()
     Map outputs = [
-      "notification.type"  : "destroyjob",
+      "notification.type"  : "serializeApplication",
```
Comment (duftler) [inline orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/serialize/SerializeApplicationTask.groovy]: Looks like the other `notification.type`'s are all lowercased.

Comment (duftler) [inline orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/serialize/SerializeApplicationTask.groovy]: Please align the `:`'s.

**Comment by duftler** [comment]
LGTM after addressing comments.

**Comment by nwwebb** [comment]
addressed

**Pull request merged**